.github/workflows/docker-publish.yml

```diff
@@ -73,59 +43,58 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
-      - name: GPU image
-        if: github.event_name == 'push'
-        uses: docker/build-push-action@v3
+      -
```

Comment:
useless blank line (nitpicking, but since we're likely to make further edits before merging...)